size_test.go

```diff
@@ -116,6 +116,8 @@ func TestRAMInBytes(t *testing.T) {
 	assertSuccessEquals(t, 32*KiB, RAMInBytes, "32K")
 	assertSuccessEquals(t, 32*KiB, RAMInBytes, "32kb")
 	assertSuccessEquals(t, 32*KiB, RAMInBytes, "32Kb")
+	assertSuccessEquals(t, 32*KiB, RAMInBytes, "32Kib")
+	assertSuccessEquals(t, 32*KiB, RAMInBytes, "32KIB")
```

Comment:
Looking at these, I wonder if it's correct that we treat `kB`, `KiB` and `KB` equal; from https://en.wikipedia.org/wiki/Kilobyte

<img width="351" alt="screen shot 2017-12-21 at 18 38 01" src="https://user-images.githubusercontent.com/1804568/34267710-248935d8-e67e-11e7-82fb-ec834a6ed4a9.png">

32 KB == 32 KiB == 32768 byte, but 32kB == 32000 byte

Reply:
Actually, wondering if I'm reading `KB` (JEDEC) correctly in that table, so please double-check 😅

Reply:
Well, that was, obviously, already there. What I added *should* be the correct behavior imho. However, I wonder how many things will break if we decide to parse it somehow around.

Also, on the other hand, there's no function here to make a difference between KB and KiB, it just depends on what function does the user of the library call.

Reply:
Yes, I noticed it was there 😞 not sure what's best; perhaps we need additional functions

Reply:
Well, I don't think this module is organized in such a way to properly support the standards. I'd rather go for a rebuild of this module (go-units2 or go-standard-units :D) and organize the functions something like ReadHumanSizeBinary which properly parses IEC and JEDEC, then ReadHumanSizeDecimal which properly parses metric data. Also organize those maps and regexes per "Metric/Decimal" and "Binary/IEC/JEDEC".

My guess is that it would break a lot of stuff because these conventions are badly used all around.

Until then, would you think this is a blocker to merge this?

Reply:
Agreed, this change doesn't make things worse, so don't see a reason to not go ahead.

Let me know if you're interested in a follow up for those things though!